strr-web/pages/application-details/[id]/index.vue

```diff
@@ -323,8 +324,79 @@ const isApprovedOrRejected: boolean =
 
 // Get Supporting Documents from the Application response
 const documents: DocumentUploadI[] = applicationDetails.documents || []
+const examinerOrHostStatus = computed(() => {
+  if (isExaminer) {
+    return application?.header.examinerStatus
+  } else {
+    return application?.header.hostStatus
+  }
+})
+const applicationStatus = application?.header.status
+const flavour = application ? getChipFlavour(examinerOrHostStatus.value || applicationStatus) : null
```

Comment:
I think here `getChipFlavour()` needs `applicationStatus` and not examiner or host status string.

Reply:
Discussed